shell/src/main/java/tachyon/shell/command/ListLineagesCommand.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.conf.TachyonConf;
+
+public final class ListLineagesCommand extends AbstractTfsShellCommand {
```

Comment:
I left them blank because there were no comments in the corresponding methods before, since I didn't write these initially so I worried that my description may not be accurate, can I just add a TODO here?